src/main/resources/map/callNumberTypes.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Dewey": 1,
+  "Library of Congress": 0,
+  "National Library of Medicine": 0,
```

Comment:
Though it will resolve to the same parsing, NLM should be type 2

Reply:
I will fix. I forgot to double check the values after creating.